templates/agenda/show.html.twig

```diff
@@ -0,0 +1,204 @@
+{% extends 'layout-with-agenda-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda' %}
+
+{% block title %}{% trans %}Agenda{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Agenda{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            {% if 4 == agenda.status %}
```

Comment:
`4` should be replaced by a constant or method on the agenda class.

Reply:
Added and used `isFinished` function to `Agenda` class.